app/assets/stylesheets/pokemon_style.css

```diff
@@ -0,0 +1,5 @@
+#notice_update {
```

Comment:
This is close but we never want to attach style to an ID. We'd much rather have it on a class which is denoted with a `.` instead of a `#`